src/cfclient/ui/tabs/FlightTab.py

```diff
@@ -328,6 +415,15 @@ def connected(self, linkURI):
         lg.add_variable("motor.m3")
         lg.add_variable("motor.m4")
 
+        sync = SyncCrazyflie(linkURI, cf=self.helper.cf)
+        self._hlCommander = PositionHlCommander(
+            sync,
```

Comment:
Don't create a SyncCrazyflie (should only be used with the "with" syntax). Instead, use self.helper.cf directly, see https://github.com/bitcraze/crazyflie-lib-python/blob/6e3f0e3c2870ab96a0276aaaf0d14bf49b758a34/cflib/positioning/position_hl_commander.py#L67-L70